screens/Settings/SettingScreen.tsx

```diff
@@ -147,7 +160,7 @@ export const SettingScreen: React.FC<
               </ListItem.Content>
               <Switch
                 value={controller.isBiometricUnlockEnabled}
```

Comment:
can we try adding testId to this switch component

Reply:
done